packages/graphql-client/documents/token-holder/tokenHolders.graphql

```diff
@@ -1,16 +1,29 @@
-query GetTopTokenHolders($after: String, $before: String, $limit: Int, $orderDirection: String ) {
-  accountBalances(orderBy: "balance", orderDirection: $orderDirection, limit: $limit, after: $after, before: $before, where: {balance_gt: 0}) {
+query GetTopTokenHolders(
+  $after: String
+  $before: String
+  $limit: Int
+  $orderDirection: String
+  $addresses: [String]
+) {
+  accountBalances(
+    orderBy: "balance"
+    orderDirection: $orderDirection
+    limit: $limit
+    after: $after
```

Comment:
Same issue as delegates query - when addresses is undefined, accountId_in: $addresses will cause the query to return no results instead of all token holders. This breaks the default behavior when no address filtering is intended.